src/main/java/cn/lanink/crystalwars/utils/inventory/ui/advanced/AdvancedInventory.java

```diff
@@ -122,8 +122,8 @@ public static void onEvent(@NotNull Event event) {
             for (InventoryAction action : transactionEvent.getTransaction().getActions()) {
                 if (action instanceof SlotChangeAction) {
                     SlotChangeAction slotChangeAction = (SlotChangeAction) action;
-                    if (slotChangeAction.getInventory() instanceof AdvancedInventory/* &&
-                            slotChangeAction.getTargetItem().getId() == 0*/) {
+                    if (slotChangeAction.getInventory() instanceof AdvancedInventory &&
+                            slotChangeAction.getTargetItem().getId() == 0) {
```

Comment:
这里会导致物品替换bug

Reply:
Hi! Much thanks for your review and suggestions! I've just made some changes to the branch [Multilanguage_support_test](https://github.com/MemoriesOfTime/CrystalWars/tree/Multilanguage_support_test)
- including removing those debug translations, fixing a bug you mentioned in [AdvancedInventory.java]. Hope it will be of help!